### DISCUSSION TITLE
Optional CW at Boost / Fav.

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#19256] [General]
In the case that the author of the tote has forgotten to set a Content Warning, it might be useful, if optional, to extend an already existing Content Warning at the Boost or to set it at all for his followers.

**Comment by Psykatte** [comment]
I feel this is an excellent idea, especially as some instances may have different CW guidelines to adhere to.

**Comment by zeborah** [comment]
I agree it would be extremely useful - but I can imagine some people would end up treating it like a quote-boost by adding snarky commentary instead of a 'traditional' CW. I can't figure out how to reconcile the benefits vs downside.

**Reply by jocelynetienne** [reply]
Agree, while at present what one can do is write a CWed-answer. 

This means that the CW text + the answer text are entirely endorsed by the answerer, without any risk of false attribution to the original poster. This addresses @Psykatte concern to still propagate the message to one's instance which doesn't have the same CW policy, and does so through a CWed answer message that can be tailored to that community. @calmeilles suggestion that a char limit could help seems to me wrong, a 10-char message is enough to be extremely harmful (aggressive *or* misleading).

**Comment by calmeilles** [comment]
I came here to see if this had been suggested or do so myself if it hadn't; so a +1 from me. There have been things (mostly politics) that I'd have boosted but hesitated to do so for lack of CW on the original.

"Show confirmation dialog before boosting" is already there and when switched on the dialogue would seem the obvious place to add such a feature if it were possible to do so.

@zeborah  Point.  At the moment the CW field shares the character limit of the toot (as far as I can see).  Perhaps it could be separately defined and fairly low (20-30 chars) to limit it's utility for snark?

**Reply by Vollans** [reply]
Agree with @calmeilles - it would be really useful as it does lead me sometimes to avoid sharing something “just in case”, whether due to the actual content offending some people, or being potential spoilers for events in progress or just happened.

**Comment by andurilos** [comment]
i wanted to add a feature request, found yours. I am totaly into adding cws to boosts.

**Comment by panomaki** [comment]
This could also be useful when boosting on instances with a different norm on when CWs are appropriate. The drawback @zeborah mentioned could possibly be mitigated by making the CW-label short (say, max 20 chars).
